library/k8s_apps_v1beta1_deployment.py

```diff
@@ -1,4 +1,5 @@
-#!/usr/bin/env python
+#!/usr/bin/python
```

Comment:
does this not mess with virtual environnment stuff? I was under the impression that `/usr/bin/env python` was preferred so that if your paths are changed by virtual environment hackery it will use the proper interpreter